zhakypadil/Habarla
# Connect db firebase

## Ticket
* N/A
## Description
* Basic setup for Cloud Firestore db. For now, only querying discounts is available. The rest will be added after completing user sing in/sign-up is done. 
## Test Plan
### Test 1: Functionality :white_check_mark:
* Click on `I am a Customer`
* Verify the list of discounts is visible (if available)

## Notes
1. I Had to update Firebase packages to the latest version manually. Had 4 critical issues when installing packages. If ever issues arise, they might happen because of this change.
2. new line in `.gitignore` - `dbDecisions.ts` is temporary. The file is just a reference for myself, will remove it later.
